src/exercises/chromium/build-rules.md

```diff
@@ -0,0 +1,58 @@
+# Build rules exercise
+
+In your Chromium build, add a new Rust target to `//ui/base/BUILD.gn` containing:
+
+```rust
+#[no_mangle]
+pub extern "C" fn hello_from_rust() {
+    println!("Hello from Rust!")
+}
+```
+**Important**: note that `no_mangle` here is considered a type of unsafety
+by the Rust compiler, so you'll need to to allow unsafe code in your
+`gn` target.
+
+Add this new Rust target as a dependency of `//ui/base:base`.
+Declare this function at the top of `ui/base/resource/resource_bundle.cc`
+(later, we'll see how this can be automated by bindings generation tools):
+```cpp
+extern "C" void hello_from_rust();
+```
+
+Call this function from somewhere in `ui/base/resource/resource_bundle.cc` -
+we suggest the top of `ResourceBundle::MaybeMangleLocalizedString`.
+Build and run Chromium, and ensure that "Hello from Rust!" is printed lots of times.
```

Comment:
Optional suggestion: Maybe running a small subset of `ui_unittests` may be preferrable?  (shorter/faster iteration and feedback loop)

Reply:
True. But it's nice to see a visual outcome of your code changes. I'll keep it as-is for now but have recommended a component debug build in the setup.